.github/workflows/website.yml

```diff
@@ -21,8 +21,11 @@ jobs:
       - uses: actions/setup-node@v2
         with:
           node-version: 16
+      - # See https://github.com/npm/cli/issues/3637
+      - run: npm i -g npm@7.20.2
+      - run: npm ci
```

Comment:
```suggestion
```
the step after this one does `npm ci` but adds in caching the dep tree on github based on the state of the package-lock.js
see: https://github.com/bahmutov/npm-install#basic

Reply:
Got it, makes sense. Removed both of these in favour of `npm i` being run from the repo root, for all workspaces.